### DISCUSSION TITLE
DIA-NN could not finish the direct-DIA analysis of phosphoproteomics raw data

### DISCUSSION THREAD

**Discussion by Senseiproteomics** [vdemichev/DiaNN#600] [Q&A]
Hi Vadim,

We tried to do direct-DIA phosphoproteomics analysis using DIA-NN (V1.8.1). However, the process stuck at the step Predicting spectra and IMs - Generating predictions. A DIA-NN outage lasted two weeks.

Attached are the settings:
![image](https://user-images.githubusercontent.com/123764948/215162575-d10ec4ec-0411-40ea-85e2-8d6dde345826.png)

Is there any way to speed up the process? For this type of job, what settings should be used in the DIA-NN GUI, and what computer specs are needed?

Thank you so much for your time and look forward to hearing from you.

Sincerely,

Green

**Comment by vdemichev** [comment]
Hi Green,

Yes, the prediction is VERY slow when DIA-NN needs to predict tens of millions of spectra. I recommend doing it in a separate pipeline step and expect it to take up to several days. It's better to limit missed cleavages to 1, limit precursor charge range and mass range. Probably also better limit the number of threads to 16.

Best,
Vadim

**Comment by Senseiproteomics** [comment]
Hi Vadim,
Thank you for your prompt response and advice. I will change the miss-cleavage to 1, the precursor charge to 2-4, and the threads to 16.
Further, to speed up the real data analysis, I'll create the phosphor-proteome prediction first, then use the generated phosphor-peptide library in the real database search. How does this make sense to you?
My sincere thanks,
Guolin
